src/main/java/no/nav/foreldrepenger/regler/uttak/fastsetteperiode/betingelser/SjekkOmTomForAlleSineKontoer.java

```diff
@@ -22,28 +23,28 @@ public SjekkOmTomForAlleSineKontoer() {
 
     @Override
     public Evaluation evaluate(FastsettePeriodeGrunnlag grunnlag) {
+        var ikkeMinsterett = !grunnlag.isSakMedMinsterett();
         var tomForAlleSineKontoer = true;
         for (var stønadskontotype : hentSøkerSineKontoer(grunnlag)) {
             for (var aktivitet : grunnlag.getAktuellPeriode().getAktiviteter()) {
                 var saldo = grunnlag.getSaldoUtregning().saldoITrekkdager(stønadskontotype, aktivitet);
-                if (saldo.merEnn0()) {
+                var gulv = ikkeMinsterett || grunnlag.getAktuellPeriode().kanTrekkeAvMinsterett() ? Trekkdager.ZERO :
```

Comment:
bedre med !misterett i stedet for "ikke" i navnet?

Reply:
Den slo meg også da jeg skrev det